doc/sphinx/source/development.rst

```diff
@@ -94,6 +94,16 @@ and leaves the diff uncommitted in the working tree.
 used in an interactive rebase session. Both scripts require "clang-format" binary
 present in PATH.
 
+.. note::
+    If the format scripts return an error similar to:
+
+    "UnicodeDecodeError: 'utf-8' codec can't decode byte 0xb3 in position 173310:
+    invalid start byte"
+
+    It is likely that the commit contains a binary file that is erroneously
+    being parsed. One solution is to break out the binary file into a separate
+    commit and later squash it once the formatting is done.
+
```

Comment:
This [SO issue](https://stackoverflow.com/questions/56649224/unicodedecodeerror-utf-8-codec-cant-decode-byte-0xb3-in-position-173310-inv/56660699#56660699) makes me think that we have files with non-ascii/non-utf encoding coming from the Windows porting effort. Did you encounter the issue mentioned here? Can you point out the files?

Reply:
yeah I had this when I added a raw YUV image file and it was showing up in the patch.

Reply:
yes for those specific file extensions, but in the future a new file might cause the same problem.